eliza.py

```diff
@@ -28,6 +29,28 @@ def InsereDicionario(string,dicionario):
     else:
         print("AVISO: Problema de sintaxe")
 
+# Dado uma string com sinonimos, associa cada palavra dessa string com sua lista de sinonimos
+lista_sinonimos = []
+def read_synons(string,dicionario):
+    sinonimos = string.split()
+    # Se não há palavras suficientes
+    if len(sinonimos) <= 1:
+        print("AVISO: Problema de sintaxe")
+        return
+    lista_sinonimos.append(sinonimos)
+    # Para cada palavra, associa ela a sua lista de sinonimos
+    for palavra in lista_sinonimos[-1]:
+        dicionario[palavra] = lista_sinonimos[-1]
+
+# Dado duas string, retorna True se elas são sinônimas, False se não são
+def are_sinonims(string1,string2):
+    l1 = language['synon'].get(string1)
+    l2 = language['synon'].get(string2)
+    if l1 is not None and l2 is not None and l1 == l2:
```

Comment:
Não precisamos desse if, é só retornar a expressão:
`return l1 is not None and l2 is not None and l1 == l2`